src/main/java/com/hazelcast/web/HazelcastInstanceLoader.java

```diff
@@ -100,32 +130,26 @@ private static Config getServerConfig(String mapName, URL configUrl, String sess
                 throw new ServletException(e);
             }
         }
-        if (sessionTTLConfig == null) {
-            sessionTTLConfig = SESSION_TTL_DEFAULT_SECONDS;
+
+        if (!isNullOrEmptyAfterTrim(instanceName)) {
```

Comment:
According to the flow described by @gAmUssA , there won't be any instance creation if an instanceName is given. Either use it if exists or fail fast during startup....
see the comment [here] (https://github.com/hazelcast/hazelcast-wm/issues/28#issuecomment-260798855) I am still pretty much worried if that causes any trouble for existing users who are gonna upgrade to latest version of hazelcast-wm.